src/api-reference/profile/search.md

```diff
@@ -0,0 +1,341 @@
+---
+title: Profile Search
```

Comment:
Hi @mvahlsing98117. Could you give me a little background on this? This looks like an entirely new API.